src/avram/model.cr

```diff
@@ -8,17 +8,11 @@ abstract class Avram::Model
   include Avram::Polymorphic
   include Avram::SchemaEnforcer
 
-  module JSONConverter(T)
-    def self.from_rs(rs : DB::ResultSet)
-      value = rs.read(JSON::PullParser?)
```

Comment:
This just moved to its own file so the `DB::Field` annotation could access it without being namespaced to `Avram::Model`